2024-11-19T23:45:42.591965+00:00 CRITICAL 'ansible_navigator.actions.run_playbook._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '77891630-9160-48eb-912a-3d4158729225', 'counter': 1, 'stdout': '\x1b[0;31mERROR! the playbook: playbooks/gathered.yml could not be found\x1b[0m', 'start_line': 0, 'end_line': 1, 'runner_ident': '7300f49d-d3a0-4b24-8e8c-f1bf0019afc8', 'created': '2024-11-19T23:45:42.590857'}
